tox.ini

```diff
@@ -33,18 +33,18 @@ commands = {posargs}
 setenv = VIRTUALENV_DIR = {envdir}
```

Comment:
This file is not actually used anywhere anymore (afaik).

Any objections to removing it?